docs/components/remote-view.md

```diff
@@ -0,0 +1,296 @@
+---
+parent: Components
+title: <RemoteView />
+---
+
+# `<RemoteView />`
+
+`<RemoteView />` is a Modular React component that enables the micro-frontend
+pattern using Modular [ESM views](https://modular.js.org/esm-views/).
+
+## Features
+
+- Build distributed UIs using the microfrontend pattern, thanks to Modular
+  [ESM Views](https://modular.js.org/esm-views/) and an
+  [ESM CDN](https://modular.js.org/esm-views/esm-cdn/)
+- Supports loading local and global CSS
+- Flexible error handling
+- Declaratively fall back to iframes
+- Compatible with any ESM CDN
+- Also supports Modular [apps](https://modular.js.org/package-types/) via
+  iframes
+
+## Getting Started
+
+Install the RemoteView package:
+
+```bash
+yarn add @modular-scripts/remote-view
+```
+
+or
+
+```bash
+npm install @modular-scripts/remote-view
+```
+
+Modular `<RemoteView>`s work by
+[dynamically importing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import)
+your Modular [ESM Views](https://modular.js.org/esm-views) and natively
+rendering them into your React tree.
+
+To achieve this, `<RemoteView>` expects to load ESM Views from an ESM CDN, which
+exposes code as a native ES Module, plus accompanying files such as static CSS
+and the `package.json`.
+
+This approach enables the microfrontend pattern, which can also be thought as
+distributed UIs.
+
+`<RemoteView />` also supports rendering non-ESM View modules by falling back to
+iframes. In this case, an iframe is rendered in the React tree, instead of a
+native React subtree.
+
+## Example
+
+Load (dynamically import) and render a set of
+[esm-cdn-hosted](https://modular.js.org/esm-views/esm-cdn/) views:
+
+```jsx
+import React, { useState } from 'react';
+import {
+  RemoteViewProvider,
+  RemoteView,
+  RemoteViewErrorBoundary,
+} from '@modular-scripts/remote-view';
+
+function MyPortal() {
+  // In practice, URLs to your remote ESM views can be fetched from a server
+  const [remoteViews, addRemoteView] = useState([
+    'https://cdn.example.com/esm-view-1',
+    'https://cdn.example.com/esm-view-2',
+  ]);
+
+  return (
+    <RemoteViewProvider>
+      {remoteViews.map((url, key) => (
+        <section key={key}>
+          <RemoteViewErrorBoundary>
```

Comment:
Ah, ok, it looks like it has a default content?

Reply:
It has 2 optional props: children and `content` - this is described in the docs. 
Most of the time it just serves to render `children`.

But yes, it also has default error fallback content. This can be seen in the demos (but is also described in the readme). It's a very simple default: `packages/remote-view/src/components/default-remote-view-error-fallback.tsx`